### DISCUSSION TITLE
Disabling the new drag-tiling-windows function

### DISCUSSION THREAD

**Discussion by awrush** [i3/i3#5153] [Q&A] [answered]
Hey everyone,

I was just wondering if it is possible to disable the new dragging option for tiling windows. I skimmed the changes but could not find any new configuration option to disable it.
I often drag windows by accident when switching tabs via mouse. Some way to revert to the previous behavior would be appreciated.

Thank you all for the great work on i3.

**Comment by orestisfl** [comment]
Just to understand, how do you accidentally drag windows? Which keybind in which application?

**Comment by awrush** [comment]
I'm using a tabbed layout. When I switch via mouseclick to another tab, the drag event sometimes occurs (probably not releasing the left button fast enough whilst dragging the cursor up slightly), which turns my layout into a split view.

**Reply by orestisfl** [reply]
I see. Just as a temporary work around you can right-click or use the scroll wheel to change focus.

**Comment by Br31zh** [comment]
Maybe adding an option to configure a margin/deadzone before actually triggering this feature, without entirely deactivating it, could help? (I have this problem too).

**Reply by scnale** [reply]
A configuration option might not be necessary. For example, I think the sensitivity is way too high for a mouse.
I'm not sure about other input methods but they might require even less sensitivity. This is the only reason I can think of to add configuration for this.

**Comment by oselcuk** [comment]
To add another data point, I frequently find myself accidentally dragging windows when I meant to be dragging browser tabs. A way to disable the mouse dragging entirely (or at least make it only work with the floating modifier) would be great

**Comment by kubami** [comment]
+1 I keep accidentally moving my windows with the mouse. I don't foresee using this feature, would love to be able to turn it off completely.

**Comment by orestisfl** [comment, marked as answer]
This is now tracked in #5155, I am locking this discussion. Please refrain from +1 comments, prefer to use emojis to vote for an issue.
